searching.rst

```diff
@@ -43,9 +43,7 @@ The various placeholders are as follows:
 
 :var:`name-like`:
   Any of
-  :var:`name`\ :path:`/`\ :glob:`*`,
-  :var:`name`\ :path:`-`\ :glob:`*`, or
-  :var:`name`,
+  :var:`name`\ :path:`/`\ :glob:`*` or :var:`name`,
```

Comment:
```suggestion
  :var:`name`\ :path:`/`\ :glob:`*` or
  :var:`name`,
```

Keeping each item on its own line reduces diff churn when the list changes. (Not perfect because of the 'or', but...)

Alternatively, move "Any of" onto this line also.

Reply:
broken back into two lines

Reply:
Thanks! :+1: